app/views/content/welcome/landing/_teaching-is-so-rewarding.html.erb

```diff
@@ -55,8 +55,8 @@
       <%= tag.p(subject_specific_quote_data.fetch(:headline), class: "pink") %>
     </div>
 
-    <%= image_pack_tag("static/images/content/hero-images/0017.jpg", alt: "A teacher supporting a pupil with work in the classroom") %>
-    <%= image_pack_tag("static/images/content/hero-images/0004.jpg", alt: "A teacher standing at the front of her classroom teaching a class") %>
+    <%= image_pack_tag("static/images/content/hero-images/teacher2.jpg", alt: "A teacher supporting a pupil with work in the classroom") %>
+    <%= image_pack_tag("static/images/content/hero-images/maths.jpg", alt: "A teacher standing at the front of her classroom teaching a class") %>
```

Comment:
I think we'd be better with one of the campaign images here rather than the photos from the TV shoot? Would it be better to have a female teacher as well?